src/Eshopworld.WorkerProcess/Configuration/CosmosDataStoreOptions.cs

```diff
@@ -22,10 +22,15 @@ public class CosmosDataStoreOptions
         /// </summary>
         public string RequestsCollection { get; set; } = "LeaseRequests";
 
+        /// <summary>
+        /// The document collection for storing distributed lock. Only required for <see cref="DistributedLock"/>.
+        /// </summary>
+        public string DistributedLocksCollection { get; set; } = "DistributedLocks";
```

Comment:
Are we opting for a single shared collection?

Reply:
No the idea is to name it `$"{DistributedLocks}_{ApplicationName}". Right now there's no concept of ApplicationName in there. I'm hoping that it comes with the common WP Cosmos DB change.

Reply:
So this value will be set by the consumer application right?